README.md

```diff
@@ -52,12 +52,16 @@ DataValues Time has been written by the Wikidata team, as [Wikimedia Germany]
 
 ## Release notes
 
-### 0.8.1 (dev)
+### 0.8.1 (2015-08-14)
 
 #### Additions
 * Added `YearMonthDayTimeParser`
```

Comment:
Well this would normally go in a 0.9 but as this also has not reached v1 yet I don't see a major issue (especially as it is a non breaker)